user/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http        import JsonResponse
+from django.views       import View
+
+from user.models        import UserInfo
+from lush.settings      import SECRET_KEY
+from lush.settings      import ALGORITHMS
+
+class LoginView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try :
+            if UserInfo.objects.filter(user_id = data['user_id']).exists():
+                user = UserInfo.objects.get(user_id = data['user_id'])
+                user_password = user.password.encode('utf-8')
+
+                if bcrypt.checkpw(data['password'].encode('utf-8'),user_password):
+                    token = jwt.encode({'id' : user.id}, SECRET_KEY, ALGORITHMS)
+                    token = token.decode('utf-8')
```

Comment:
```suggestion
                    token = jwt.encode({'id' : user.id}, SECRET_KEY, ALGORITHMS).decode('utf-8')
```
굳이 두 번의 절차를 거칠 필요가 없다고 판단됩니다.